index.ts

```diff
@@ -1,6 +1,6 @@
 // TODO: this form is deprecated, we should not rename these types. Should change in next mayor update.
-export { AsyncEvaluator, IAsyncEvaluatorConfig as AsyncEvaluatorConfig } from './lib/evaluators/AsyncEvaluator';
-export { SyncEvaluator, ISyncEvaluatorConfig as SyncEvaluatorConfig } from './lib/evaluators/SyncEvaluator';
+export { AsyncEvaluator, IAsyncEvaluatorContext as AsyncEvaluatorConfig } from './lib/evaluators/AsyncEvaluator';
+export { SyncEvaluator, ISyncEvaluatorContext as SyncEvaluatorConfig } from './lib/evaluators/SyncEvaluator';
```

Comment:
We need to remain backwards-compatible, so make sure to re-export them as `IAsyncEvaluatorConfig` and `ISyncEvaluatorConfig` as well, but add a deprecation notice (with TODO to remove in next major version).

Reply:
I will, Thank you for noticing.

Reply:
I will, thank you!